RTech.CoffeeMachine.Api.Tests/Usings.cs

```diff
@@ -0,0 +1 @@
+﻿global using Xunit;
```

Comment:
I personally use xUnit, & have used nUnit in the past (back then, nUnit had better parallel fixture support compared to xUnit).